### ISSUE TITLE
cmake caches PYTHON_DEST after first run of cmake-gui

### ISSUE THREAD

**Issue by markus-jehl** [UCL/STIR#980]
If you run cmake-gui (or ccmake) first, and then change CMAKE_INSTALL_PREFIX, then it keeps PYTHON_DEST (and STIR_CONFIG or so) pointing to the original, i.e. /usr/local on Linux.

**Comment by robbietuk** [comment]
This is true. `PYTHON_DEST` shouldn't be directly tided to `${CMAKE_INSTALL_PREFIX}/python` in case a custom `PYTHON_DEST` is required. 

Personally, I set the `CMAKE_INSTALL_PREFIX`, configure and then enable python. This sets the `PYTHON_DEST` to `${CMAKE_INSTALL_PREFIX}/python`.

**Comment by KrisThielemans** [comment]
strategy to circumvent this is shown at https://github.com/SyneRBI/SIRF/pull/160#issuecomment-381893762

**Comment by robbietuk** [comment]
This is true for `STIR_CONFIG_DIR` too. It defaults to `/usr/local/share/stir/config`
